go.mod

```diff
@@ -5,17 +5,17 @@ go 1.22
 require (
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
 	github.com/go-logr/logr v1.4.1
-	github.com/golang/mock v1.6.0
 	github.com/google/go-cmp v0.6.0
 	github.com/google/gofuzz v1.2.0
 	github.com/google/uuid v1.4.0
-	github.com/gophercloud/gophercloud v1.11.0
+	github.com/gophercloud/gophercloud v1.7.0
```

Comment:
This seems unrelated and unnecessary. Please keep it as it was

Reply:
`github.com/gophercloud/gophercloud v1.11.0` has been restored